manuscript.qmd

```diff
@@ -378,13 +374,13 @@ SE data science ecosystem distinguishes our TreeSE-based approach from phyloseq,
 which is another popular Bioconductor data container originally 
 developed with a focus on 16S amplicon data analysis [@mcmurdie2012]. 
 TreeSE is a superset that can accommodate all elements of a phyloseq object,
-while also offering improved support for multi-modal data and often substantial
+while also offering support for multi-modal data and often substantial
```

Comment:
Removing "improved" is actually true. phyloseq do not have support for multimodal data. Good.